### ISSUE TITLE
A11y_SQLAzureDataStudio_SQLServerBigDataCluster_Summary_ScreenReader:The table name is not announced

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#9299]
[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

#A11y_SQLAzureDataStudioBenchmark;#A11yMAS;#A11ySev2;#A11yTCS;#SQL Azure Data Studio;#Benchmark;#DesktopApp;#Win32;#MAS1.3.1;#JAWS;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.16.0-insider
OS: Windows_NT x64 10.0.18363

**Additional Details:**
Screen reader:JAWS
MAS References:MAS1.3.1

**Repro Steps:**
1. Launch Azure Data Studio.
2. Select the Connections icon (if not already selected) from left bar.
3. Click on the ellipses '...' and select "New Deployment".
4. Select "SQL Server Big Data Cluster" card under deployment options.
5. Select the version & checkbox.
6. Navigate till Selet button & hit enter.
7. Select Configuration profile --> Update Azure settings --> Update Cluster settings --> Update Service settings & hit Next.
8. Launch screen reader as JAWS and navigate to the table under "Storage settings" button.

**Actual:**
On navigating using down arrow key to the table under the "Storage settings" button the row and column value are announced but not the table name.

Issue not observed with NVDA

**Expected:**
The table name should be announced while the row and column value is announced.

**User Impact:**
Screen reader users will know what is the purpose of the table they are reading.

**Attachment link for Reference :**
[A11y_SQLAzureDataStudio_SQLServerBigDataCluster_Summary_ScreenReader9299.zip](https://github.com/microsoft/azuredatastudio/files/4322616/A11y_SQLAzureDataStudio_SQLServerBigDataCluster_Summary_ScreenReader9299.zip)

**Comment by BhanuPrakashGit** [comment]
As "SQL Server Big Data Cluster" feature has been removed.
**GithubTags:**#Closed;

**Comment by mstechie** [comment]
GitHubTags:#SQLADS-Win32-Feb2020;
